eng/common/scripts/Update-DocsMsMetadata.ps1

```diff
@@ -230,17 +230,39 @@ function UpdateDocsMsMetadataForPackage($packageInfoJsonLocation) {
   Set-Content -Path $readmeLocation -Value $outputReadmeContent
 }
 
-# For daily update and release, validate DocsMS publishing using the language-specific validation function
-if ($ValidateDocsMsPackagesFn -and (Test-Path "Function:$ValidateDocsMsPackagesFn")) {
-  Write-Host "Validating the packages..."
+$allSucceeded = $true
+foreach ($packageInfoLocation in $PackageInfoJsonLocations) {
 
-  $packageInfos = @($PackageInfoJsonLocations | ForEach-Object { GetPackageInfoJson $_ })
+  if ($ValidateDocsMsPackagesFn -and (Test-Path "Function:$ValidateDocsMsPackagesFn")) {
+    Write-Host "Validating the packages..."
 
-  &$ValidateDocsMsPackagesFn -PackageInfos $packageInfos -PackageSourceOverride $PackageSourceOverride -DocValidationImageId $DocValidationImageId -DocRepoLocation $DocRepoLocation
-}
+    $packageInfo =  GetPackageInfoJson $packageInfoLocation
+    # "Validate-${Language}-DocMsPackages"
```

Comment:
Should this be removed?

Reply:
I think this comment belongs in more places as it helps the user figure out what `$ValidateDocsMsPackagesFn` is without having to go to `common.ps1`, look up the string value, and then search from there.

Reply:
I thought this was a call to a function when skimming the code, not necessarily a comment. Maybe you can elaborate just a bit e.g. `# This calls a language-specific function named "Validate-${Language}-DocMsPackages" under the hood.`